.env.development

```diff
@@ -7,4 +7,4 @@ NEXT_PUBLIC_API_NEW=http://127.0.0.1:4010/mock
 NEXT_PUBLIC_BASE=
 NEXT_PUBLIC_CONFIG_API=
 NEXT_PUBLIC_COUNTRY_LEADER_BOARD_DISABLED=false
-NEXT_PUBLIC_SERVER_CONFIG_DISABLED=true
+NEXT_PUBLIC_SERVER_CONFIG_DISABLED=
```

Comment:
I don't think this change should be in this PR

Reply:
I use this for local development , I will remove that or make a new pr.